[2024-11-11T08:25:23.160+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-11T08:25:23.206+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_csv manual__2024-11-11T08:24:58.618194+00:00 [queued]>
[2024-11-11T08:25:23.218+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_csv manual__2024-11-11T08:24:58.618194+00:00 [queued]>
[2024-11-11T08:25:23.219+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-11T08:25:23.236+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): fetch_data_from_csv> on 2024-11-11 08:24:58.618194+00:00
[2024-11-11T08:25:23.242+0000] {standard_task_runner.py:64} INFO - Started process 298 to run task
[2024-11-11T08:25:23.246+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'fetch_data_from_csv', 'manual__2024-11-11T08:24:58.618194+00:00', '--job-id', '666', '--raw', '--subdir', 'DAGS_FOLDER/pipline.py', '--cfg-path', '/tmp/tmpdigf0xfh']
[2024-11-11T08:25:23.249+0000] {standard_task_runner.py:91} INFO - Job 666: Subtask fetch_data_from_csv
[2024-11-11T08:25:23.317+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.fetch_data_from_csv manual__2024-11-11T08:24:58.618194+00:00 [running]> on host 85c9117c3d09
[2024-11-11T08:25:23.527+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='fetch_data_from_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-11-11T08:24:58.618194+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-11T08:24:58.618194+00:00'
[2024-11-11T08:25:23.531+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-11T08:25:23.534+0000] {logging_mixin.py:188} INFO - Error reading CSV file: [Errno 2] No such file or directory: 'C:/Users/mohammed/Desktop/Airflow/Data-Engineering-Pipeline/***/data_set/journals.csv'
[2024-11-11T08:25:23.536+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-11T08:25:23.539+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-11T08:25:23.574+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=fetch_data_from_csv, run_id=manual__2024-11-11T08:24:58.618194+00:00, execution_date=20241111T082458, start_date=20241111T082523, end_date=20241111T082523
[2024-11-11T08:25:23.660+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-11-11T08:25:23.715+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-11T08:25:23.730+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
